index.js

```diff
@@ -41,87 +43,79 @@ const displayGuestScore = (score) => (guestPoint.textContent = score);
 
 const showMessage = (msgEl, msgVal) => (msgEl.textContent = msgVal);
 
+function showResetButton() {
+  startButton.style.display = "none";
+  resetButton.style.display = "block";
+}
 function eachRoundScoreDisplay (roundVal, hScore, gScore) {
   let rVal = roundVal;
-  if (rVal == 1){
+  if (rVal === 1){
     homeRound1Score.textContent = hScore;
     guestRound1Score.textContent = gScore;
   }
-  if (rVal == 2){
+  if (rVal === 2){
     homeRound2Score.textContent = hScore;
     guestRound2Score.textContent = gScore;
   }
-  if (rVal == 3){
+  if (rVal === 3){
     homeRound3Score.textContent = hScore;
     guestRound3Score.textContent = gScore;
   }
-  if (rVal == 4){
+  if (rVal === 4){
     homeRound4Score.textContent = hScore;
     guestRound4Score.textContent = gScore;
     calculateFinalScore();
-  }  
+  } 
 }
```

Comment:
- [ ] YOu can optimize this function to a dynamic one as well. It'll become huge if we have more numbers in it.

Reply:
Resolved